ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml

```diff
@@ -0,0 +1,124 @@
+base_images:
+  builder-go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  inputs:
+    builder-go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux-amd64
+  to: backplane-operator
+promotion:
+  disabled: true
+  name: "1.0"
+  namespace: open-cluster-management
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: sonar-pre-submit
+  commands: |
+    export SELF="make -f Makefile.prow"
+    make -f Makefile.prow sonar/go/prow
+  container:
+    from: src
+  secrets:
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
+- as: sonar-post-submit
+  commands: |
+    export SELF="make -f Makefile.prow"
+    make -f Makefile.prow sonar/go/prow
+  container:
+    from: src
+  postsubmit: true
+  secrets:
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
+- as: e2e-cluster
+  steps:
+    dependencies:
+      COMPONENT_IMAGE_REF: backplane-operator
+    env:
+      CLUSTERPOOL_GROUP_NAME: acm-cicd-prow
+      CLUSTERPOOL_HOST_NAMESPACE: cicd
+      CLUSTERPOOL_HOST_PROW_KUBE_SECRET: acm-cicd-clusterpool-shared
```

Comment:
You need to set up your own clusterpool secret for your squad and not use the cicd clusterpool secret.

Reply:
@ray-harris Am I able to just remove the e2e-cluster and e2e-kind tests? I noticed that my squad does not currently use them for any of the other repos, but that was included in David's example.

Reply:
Yes, you can remove them. The examples are not just copy/paste. You need to know what applies to your repo and then how to adapt it.